sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -125,9 +127,7 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         self._current_ctxt_manager: Optional[_SuppressionContextManager] = None
-
-        # TODO: Once we have additional supported versions, we should add a way to specify the version.
-        self._schema_version = OpenTelemetrySchema.get_latest_version()
+        self._schema_version = kwargs.pop("schema_version", _OpenTelemetrySchemaVersion.V1_23_1)
```

Comment:
can this cause some side effects for folks, because now its defaults to a specific version versus latest? Also why default to this version?

Reply:
Nothing will change here, as this is just a more explicit way of specifying the same thing, since get_latest_version() just returns `OpenTelemetrySchemaVersion.V1_23_1` anyway. We are defaulting to 1.23.1 since that's the schema whose attributes we are currently mapping to.